### PR TITLE
Show Backup Progress

### PR DESCRIPTION
**Pull request by Hirobreak**
- Showing backup progress in a snackbar

### CODE REVIEW

File: email_mailbox/src/components/MainErrorBoundary.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const HighOrderBoundarie = AppComponent =>
       hasError: false,
       errorTitle: '',
       errorInfo: '',
-      counter: 11
+      counter: 200
```
Comment (javiteri95) [inline email_mailbox/src/components/MainErrorBoundary.js:15]: counter 200 (?)

File: email_mailbox/src/lang/es.json [hunk 2]
```diff
@@ -109,6 +109,28 @@
       }
     }
   },
+  "backup": {
+    "starting_backup": "Starting Backup...",
+    "saving_account": "Saving Account...",
+    "exporting_contacts": "Exporting Contacts...",
+    "saving_contacts": "Saving Contacts...",
+    "exporting_labels": "Exporting Labels...",
+    "saving_labels": "Saving Labels...",
```
Comment (javiteri95) [inline email_mailbox/src/lang/es.json:118]: Not sure, but even es.json have to be in english?

File: electron_app/src/database/DBEexporter.js [hunk 3]
```diff
@@ -832,31 +847,184 @@ const exportEncryptDatabaseToFile = async ({ outputPath, accountObj }) => {
...
+  handleProgressCallback(
+    exportProgress,
+    'exporting_domains',
+    `${recipientId}@${domain || APP_DOMAIN}`,
+    progressCallback
+  );
+
   const customDomains = await exportCustomDomainsTable(accountId);
+
+  exportProgress += 100 / PROGRESS_TOTAL_STEPS;
```
Comment (danieltigse) [inline electron_app/src/database/DBEexporter.js:1011]: encapsulate this code 🙏🏻

File: email_mailbox/src/components/MainErrorBoundary.js (hunk 1 above)
Comment (danieltigse) [reply email_mailbox/src/components/MainErrorBoundary.js:15]: actually you change it from 1100 to 11

File: email_mailbox/src/lang/es.json (hunk 2 above)
Comment (danieltigse) [reply email_mailbox/src/lang/es.json:118]: No. It needs translations.

File: electron_app/src/ipc/backup.js [hunk 4]
```diff
@@ -74,8 +108,10 @@ const doExportBackupUnencrypted = async params => {
     }
     return backupSize;
   } catch (error) {
+    console.error(error);
```
Comment (javiteri95) [inline electron_app/src/ipc/backup.js:111]: merge the PR #1467, and then change this to logger.error

File: electron_app/src/ipc/backup.js [hunk 5]
```diff
@@ -178,60 +214,134 @@ ipc.answerRenderer('restore-backup-encrypted', async params => {
...
+    const timeDiff = nextDate.diff(today);
+    autoBackupsTime.push({
+      username: account.recipientId,
+      accountId,
+      triggerTimer: timeDiff <= 0 ? 1 : timeDiff
+    });
+
+    backupDone();
+  } catch (backupErr) {
+    log(
```
Comment (javiteri95) [inline electron_app/src/ipc/backup.js:308]: same comment

Comment (Hirobreak) [reply electron_app/src/ipc/backup.js:307]: This log function wraps all console logs in the file, I even use it in PR #1467

**Pull request merged**